src/game_interpreter.cpp

```diff
@@ -142,11 +142,18 @@ void Game_Interpreter::SetContinuation(Game_Interpreter::ContinuationFunction fu
 	continuation = func;
 }
 
+bool Game_Interpreter::ReachedLoopLimit() const {
+	return loop_count >= 10000;
```

Comment:
This should be a constant here already. I have seen this in another commit.

Reply:
Thats fixed in #1713